pkg/controllers/controllers_suite_test.go

```diff
@@ -96,8 +96,9 @@ var _ = BeforeSuite(func(done Done) {
 	}
 
 	stopCh = make(chan struct{})
```

Comment:
stopCh needs to be replaced with a context that is closed  in AfterSuite, as is the context will be closed immediately. Did tests pass?

Reply:
I had assumed there was some CI/CD. Are developers expected to manually run tests and report if they pass?

Reply:
I did update the code to handle the closing in AfterSuite though.